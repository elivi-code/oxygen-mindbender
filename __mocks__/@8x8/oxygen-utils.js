export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
