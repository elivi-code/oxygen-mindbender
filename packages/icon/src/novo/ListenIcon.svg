<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M8.25 8.25C8.25 4.51805 11.301 1.5 15.0249 1.5C18.7214 1.5 21.75 4.49588 21.75 8.2005C21.75 10.0145 21.0649 11.7616 19.8319 13.0921L18.0481 15.0169C17.3973 15.7192 17.0357 16.6412 17.0357 17.5986V18.1071C17.0357 20.5332 15.069 22.5 12.6429 22.5C10.2167 22.5 8.25 20.5332 8.25 18.1071V16.2391C8.25 15.8249 8.58579 15.4891 9 15.4891C9.41421 15.4891 9.75 15.8249 9.75 16.2391V18.1071C9.75 19.7048 11.0452 21 12.6429 21C14.2405 21 15.5357 19.7048 15.5357 18.1071V17.5986C15.5357 16.2631 16.0401 14.9769 16.9479 13.9973L18.7317 12.0725C19.7077 11.0193 20.25 9.63639 20.25 8.2005C20.25 5.33237 17.9011 3 15.0249 3C12.1213 3 9.75 5.35454 9.75 8.25V8.6087C9.75 9.02291 9.41421 9.3587 9 9.3587C8.58579 9.3587 8.25 9.02291 8.25 8.6087V8.25Z" fill="#292929"/>
<path d="M2.25 12.375C2.25 11.9608 2.58579 11.625 3 11.625H6C6.41421 11.625 6.75 11.9608 6.75 12.375C6.75 12.7892 6.41421 13.125 6 13.125H3C2.58579 13.125 2.25 12.7892 2.25 12.375Z" fill="#292929"/>
<path d="M3.2196 15.8408C2.92671 16.1337 2.92671 16.6085 3.2196 16.9014C3.51249 17.1943 3.98737 17.1943 4.28026 16.9014L5.65158 15.5301C5.94447 15.2372 5.94447 14.7623 5.65158 14.4694C5.35869 14.1765 4.88381 14.1765 4.59092 14.4694L3.2196 15.8408Z" fill="#292929"/>
<path d="M3.2196 7.71967C2.92671 8.01256 2.92671 8.48744 3.2196 8.78033L4.59092 10.1517C4.88381 10.4445 5.35869 10.4445 5.65158 10.1517C5.94447 9.85876 5.94447 9.38388 5.65158 9.09099L4.28026 7.71967C3.98737 7.42678 3.51249 7.42678 3.2196 7.71967Z" fill="#292929"/>
<path d="M15 4.5C12.9289 4.5 11.25 6.17893 11.25 8.25V9.50622C11.25 10.5356 12.0844 11.37 13.1138 11.37C13.3588 11.37 13.5338 11.6074 13.4613 11.8415L12.786 14.0204L11.6635 17.3878C11.5325 17.7808 11.7449 18.2055 12.1378 18.3365C12.5308 18.4675 12.9555 18.2551 13.0865 17.8622L14.214 14.4796L14.894 12.2855C15.2657 11.0862 14.3693 9.87 13.1138 9.87C12.9129 9.87 12.75 9.70713 12.75 9.50622V8.25C12.75 7.00736 13.7574 6 15 6C16.2426 6 17.25 7.00736 17.25 8.25V8.31C17.25 8.72421 17.5858 9.06 18 9.06C18.4142 9.06 18.75 8.72421 18.75 8.31V8.25C18.75 6.17893 17.0711 4.5 15 4.5Z" fill="#292929"/>
</svg>
