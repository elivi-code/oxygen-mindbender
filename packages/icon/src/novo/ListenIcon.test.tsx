/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import ListenIcon from './ListenIcon';

describe('<ListenIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<ListenIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
