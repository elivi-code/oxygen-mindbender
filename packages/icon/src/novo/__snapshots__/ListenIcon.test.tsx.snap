// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListenIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <path
      d="M8.25 8.25c0-3.732 3.051-6.75 6.7749-6.75 3.6965 0 6.7251 2.9959 6.7251 6.7005a7.1965 7.1965 0 0 1-1.9181 4.8916l-1.7838 1.9248a3.7982 3.7982 0 0 0-1.0124 2.5817v.5085c0 2.4261-1.9667 4.3929-4.3928 4.3929-2.4262 0-4.3929-1.9668-4.3929-4.3929v-1.868a.75.75 0 0 1 1.5 0v1.868C9.75 19.7048 11.0452 21 12.6429 21c1.5976 0 2.8928-1.2952 2.8928-2.8929v-.5085a5.2982 5.2982 0 0 1 1.4122-3.6013l1.7838-1.9248a5.6965 5.6965 0 0 0 1.5183-3.872C20.25 5.3324 17.9011 3 15.0249 3 12.1213 3 9.75 5.3545 9.75 8.25v.3587a.75.75 0 0 1-1.5 0V8.25ZM2.25 12.375a.75.75 0 0 1 .75-.75h3a.75.75 0 0 1 0 1.5H3a.75.75 0 0 1-.75-.75ZM3.2196 15.8408a.75.75 0 0 0 1.0607 1.0606l1.3713-1.3713a.75.75 0 0 0-1.0607-1.0607l-1.3713 1.3714ZM3.2196 7.7197a.75.75 0 0 0 0 1.0606l1.3713 1.3714A.75.75 0 0 0 5.6516 9.091L4.2803 7.7197a.75.75 0 0 0-1.0607 0Z"
    />
    <path
      d="M15 4.5c-2.0711 0-3.75 1.679-3.75 3.75v1.2562c0 1.0294.8344 1.8638 1.8638 1.8638.245 0 .42.2374.3475.4715l-.6753 2.1789-1.1225 3.3674a.75.75 0 0 0 1.423.4744l1.1275-3.3826.68-2.1941c.3717-1.1993-.5247-2.4155-1.7802-2.4155a.3638.3638 0 0 1-.3638-.3638V8.25C12.75 7.0074 13.7574 6 15 6s2.25 1.0074 2.25 2.25v.06a.75.75 0 0 0 1.5 0v-.06c0-2.071-1.6789-3.75-3.75-3.75Z"
    />
  </g>
</svg>
`;
